testsuite/domain/src/test/resources/host-configs/host-slave.xml

```diff
@@ -141,6 +141,8 @@
         </server>
         <server name="reload-two" group="reload-test-group"  auto-start="false">
             <jvm name="default" />
+            <!-- this is for ServerRestartRequiredTestCase.testSocketBinding() unless the socket-binding-group is actually used for something it is ignored on the slaves, this is a quick fix before correcting the test -->
+            <socket-bindings socket-binding-group="reload-sockets" port-offset="1550"/>
```

Comment:
That chain is what's there when the domain is first started; i.e. it's what the xml files show. It's possible that it's no longer there by the time this test runs, due to some other change made by a test after the domain starts. But I don't see that being done in ServerRestartRequiredTestCase. I know ServerRestartRequiredTestCase will fail even if it is run by itself with -Dtest (i.e. not part of a larger test suite with other tests that may alter the config).